plugins/plugins.go

```diff
@@ -20,8 +20,9 @@ import (
 
 const timeoutEnv = "BUFFALO_PLUGIN_TIMEOUT"
 
+var t = time.Second * 2
+
 func timeout() time.Duration {
-	t := time.Second
```

Comment:
Curious why the change in scope here?

Reply:
Because of the oncer, the timeout is set once to the `BUFFALO_PLUGIN_TIMEOUT` value. Then for each call to `timeout()` after that, `t` is always set to the default value.